Tests/SwiftFormatTests/PrettyPrint/AccessorTests.swift

```diff
@@ -81,6 +81,82 @@ final class AccessorTests: PrettyPrintTestCase {
     assertPrettyPrintEqual(input: input, expected: expected, linelength: 45)
   }
 
+  func testBasicAccessorsWithSelections() {
```

Comment:
For all these test cases: Unless you’re explicitly testing that we handle multiple selection ranges, I would split them up into separate test cases. In my experience that makes it a lot easier to debug if one of the tests fail + you know what exactly the test is testing.

Reply:
I can move these test cases into `SelectionTests.swift` (which I've renamed locally from `Selections.swift`).